docs/code-examples/point3d_random_v2.cpp

```diff
@@ -0,0 +1,34 @@
+// Log some random points with color and radii.
+
+#include <rerun.hpp>
+
+#include <random>
+
+namespace rr = rerun;
+
+int main() {
+    auto rr_stream = rr::RecordingStream("points3d_random");
+    rr_stream.connect("127.0.0.1:9876");
+
+    std::default_random_engine gen;
+    std::uniform_real_distribution<float> dist_pos(-5.0, 5.0);
+    std::uniform_real_distribution<float> dist_radius(0.1, 1.0);
+    std::uniform_int_distribution<uint8_t> dist_color(0, 255);
+
+    std::vector<rr::components::Point3D> points3d(10);
+    std::generate(points3d.begin(), points3d.end(), [&] {
```

Comment:
doesn't compile on ci yet.

Ahhh C++ where you need to test each compiler individually. I bet my std lib impl leaked the necessary header in and the one on CI doesn't